projects/www/src/app/components/docs/code-example.component.ts

```diff
@@ -27,8 +27,9 @@ import { Component, Input } from '@angular/core';
       }
 
       .body {
-        padding: 0 0px;
+        padding: 16px 20px;
         overflow-x: wrap;
+        background-color: rgb(36, 41, 46);
```

Comment:
Thanks!
I think we can apply this to `pre` tags, to also cover the code samples that don't use this component.
E.g. for the installation commands at https://deploy-preview-4812--ngrx-site-v19.netlify.app/guide/entity/install

This also removes the need to add the background color here.